(dp1
S'output'
p2
S"<type 'exceptions.TypeError'> not enough arguments for format string"
p3
sS'layer'
p4
S'/home/abdulhalim/Desktop/web2py/applications/motor/controllers/default.py'
p5
sS'code'
p6
S'<code object <module> at 0x7f2f61bb41b0, file "/home/abdulhalim/Desktop/web2py/applications/motor/controllers/default.py", line 2>'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/home/abdulhalim/Desktop/web2py/gluon/restricted.py", line 219, in restricted\n    exec(ccode, environment)\n  File "/home/abdulhalim/Desktop/web2py/applications/motor/controllers/default.py", line 137, in <module>\n  File "/home/abdulhalim/Desktop/web2py/gluon/globals.py", line 409, in <lambda>\n    self._caller = lambda f: f()\n  File "/home/abdulhalim/Desktop/web2py/gluon/tools.py", line 3964, in f\n    return action(*a, **b)\n  File "/home/abdulhalim/Desktop/web2py/applications/motor/controllers/default.py", line 49, in add_device\n    db.executesql("INSERT INTO Status (device_ref_id, last_ping, server_time) VALUES(%s, %s, %s)", device_ref_id, time, time)\n  File "/home/abdulhalim/Desktop/web2py/gluon/packages/dal/pydal/base.py", line 767, in executesql\n    adapter.execute(query, placeholders)\n  File "/home/abdulhalim/Desktop/web2py/gluon/packages/dal/pydal/adapters/__init__.py", line 67, in wrap\n    return f(*args, **kwargs)\n  File "/home/abdulhalim/Desktop/web2py/gluon/packages/dal/pydal/adapters/base.py", line 410, in execute\n    rv = self.cursor.execute(command, *args[1:], **kwargs)\n  File "/home/abdulhalim/Desktop/web2py/gluon/contrib/pymysql/cursors.py", line 164, in execute\n    query = self.mogrify(query, args)\n  File "/home/abdulhalim/Desktop/web2py/gluon/contrib/pymysql/cursors.py", line 143, in mogrify\n    query = query % self._escape_args(args, conn)\nTypeError: not enough arguments for format string\n'
p11
s.